test/conformance/chainsaw/validate/clusterpolicy/standard/apicalls/default/policy.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: default
+spec:
+  validationFailureAction: Enforce
+  rules:
+  - name: limit-lb-svc
+    match:
+      any:
+      - resources:
+          kinds:
+          - Service
```

Comment:
The policy matches on services but the resources created are pods.

Reply:
I will change it